src/ess/livedata/handlers/detector_view.py

```diff
@@ -110,6 +114,10 @@ def accumulate(
         for roi_state in self._rois.values():
             roi_state.add_data(raw)
 
+        # Ratemeter: accumulate event counts
+        self._counts_total += raw.bins.size().sum().value
```

Comment:
So is the assumption here that the weights are always 1?
I guess in the nexus or kafka formats, weights are always 1 (because they record time-stamps), so you probably can never have raw data with weights.

Reply:
Correct.